cves/CVE-2013-2135.yml

```diff
@@ -45,7 +51,14 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
It might be helpful to include in the description that this issue is triggered by crafted requests containing OGNL code

Reply:
Will update to include reference to OGNL.